SpatialGDK/Source/SpatialGDK/Public/Interop/RPCs/RPC_RingBufferWithACK_Receiver.h

```diff
@@ -102,15 +102,15 @@ class MonotonicRingBufferWithACKReceiver : public TRPCBufferReceiver<PayloadType
 		for (auto Iterator = this->ReceivedRPCs.CreateIterator(); Iterator; ++Iterator)
 		{
 			Worker_EntityId EntityId = Iterator->Key;
-			if (!CanExtract(EntityId))
-			{
-				continue;
-			}
 			ReceiverState& State = ReceiverStates.FindChecked(EntityId);
 			auto& Payloads = Iterator->Value;
 			uint32 ProcessedRPCs = 0;
 			for (const auto& Payload : Payloads)
 			{
+				if (!CanExtract(EntityId))
```

Comment:
Worth adding a code comment why this was moved here

Reply:
Will do, thank you for the suggestion :) !